cfme/fixtures/v2v_fixtures.py

```diff
@@ -197,7 +199,10 @@ def set_conversion_instance_for_osp(appliance, transformation_method, osp_provid
         pytest.skip("Failed to delete all conversion hosts:".format(delete_hosts.output))
 
     try:
-        conversion_instances = osp_provider.data.get('conversion_instances', [])
+        if transformation_method == "vddk":
```

Comment:
Question: Thoughts on having `transformation_method` have a default value? Also since this variable is equivalent to the key, why not pass it into the dict? i.e. 
```
conversion_instances = osp_provider.data['conversion_instances'][transformation_method]
```

Reply:
Good idea :+1: